packages/rum-core/src/domain/rumEventsCollection/action/actionCollection.ts

```diff
@@ -76,6 +74,10 @@ function processAction(
             count: action.counts.resourceCount,
           },
         },
+        _dd: {
+          target: action.target,
+          position: action.position,
+        },
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏From schema, it seems to be `_dd.action.target` and `_dd.action.position` but all your updates are on `_dd.target` and `_dd.position`. Is it a mistake or am I missing something?

Reply:
It's a mistake, thanks :)